packages/mobile/src/identity/reducer.ts

```diff
@@ -384,3 +390,18 @@ export const providerAddressesSelector = ({ identity: { addressToDisplayName } }
     .filter(([_, info]) => info?.isProviderAddress)
     .map(([address, _]) => address)
 }
+
+export const identifierToe164NumberSelector = createSelector(
```

Comment:
Nit: camel case appears a but off...shouldn't it be `identifierToE164PhoneNumber`?

Reply:
Btw I noticed this in a couple of other places but didn't want to spam with a bunch of comments